src/Sylius/Bundle/UiBundle/Registry/TemplateBlock.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\UiBundle\Registry;
+
+final class TemplateBlock
+{
+    /** @var string */
+    private $name;
+
+    /** @var string */
+    private $template;
+
+    /** @var int */
+    private $priority;
+
+    /** @var bool */
+    private $enabled;
+
+    public function __construct(
+        string $name,
+        string $template,
+        int $priority,
+        bool $enabled
+    ) {
+        $this->name = $name;
+        $this->template = $template;
+        $this->priority = $priority;
+        $this->enabled = $enabled;
+    }
+
+    public function name(): string
```

Comment:
Is there a reason you didn't choose to add `get` to these methods. You know by convention or consistency or something :)

Reply:
It's the style for DTOs I'm used to in the internal projects, but I'm gonna change it for OS for consistency :+1: